docs/_documentation/faq.md

```diff
@@ -110,7 +110,7 @@ eval "$(docker-machine env default)"
 
 When there are no events available for an event-type because they've expired, then `newest_available_offset` will be smaller than `oldest_available_offset`. Because Nakadi has exclusive offset handling, it shows the offset of the last message in `newest_available_offset`.
 
-#### Is there a way to make publishing batches of events atomic?
+#### Is there a way to make publishing batches of events atomic?
```

Comment:
It was no break space &nbsp; instead of normal space. Markdown parser didn't recognize it as a header because of this and show it as a raw text with ####.